src/layouts/hass-router-page.ts

```diff
@@ -256,10 +254,12 @@ export class HassRouterPage extends ReactiveElement {
   }
 
   protected createLoadingScreen() {
+    import("./hass-loading-screen");
```

Comment:
Does this not have to be awaited?  Does lit pick up properties being set downstream (before the element is defined)?

Reply:
Yes, it will be upgraded ones defined